modules/boulder/src/test/ts/atomic/api/FieldSchemaTest.ts

```diff
@@ -2,11 +2,12 @@ import { Assert, UnitTest } from '@ephox/bedrock-client';
 import { Obj, Optional } from '@ephox/katamari';
 import { KAssert } from '@ephox/katamari-assertions';
 import * as FieldSchema from 'ephox/boulder/api/FieldSchema';
-import * as ValueSchema from 'ephox/boulder/api/ValueSchema';
-import { FieldProcessorAdt } from 'ephox/boulder/core/ValueProcessor';
+import { FieldProcessor } from 'ephox/boulder/api/Main';
+import * as ValueSchema from 'ephox/boulder/api/StructureSchema';
```

Comment:
```suggestion
import * as ValueSchema from 'ephox/boulder/api/StructureSchema';
import { FieldProcessor } from 'ephox/boulder/core/FieldProcessor';
```
We shouldn't import from `Main`.

Reply:
Also the `ValueSchema` import needs renaming.